Clarify instructions for requesting new custom ZHA device handlers (#44883)
home-assistant/home-assistant.github.io: source/_integrations/zha.markdown

1. Try to re-pair/re-join the device several times.
2. Review the troubleshooting sections.
3. Search for similar situations in the Home Assistant [community forum or Discord chat server](https://www.home-assistant.io/help/).

> 4. Still not working? You may need a new custom ZHA device handler. This device handler will add exception handling code to ZHA in order to work around device-specific issues. Read instructions below on now to submit a “device support request” for a new custom ZHA device handler if there are not already an open request for your specific device model.

Reviewer: Combining the robot's feedback + simplify:

```suggestion
4. Still not working? You might need a new custom ZHA device handler. This device handler adds exception handling code to ZHA to work around device-specific issues.
   To submit a device support request for a new custom ZHA device handler, see [ZHA device handlers](#zha-device-handlers).
```
